src/manager/agent.config.ts

```diff
@@ -62,8 +62,8 @@ export class AgentConfig {
      * Sets up the agent config, using smart defaults from the AgentCreateDto
      */
     constructor(agentDto: AgentCreateDto) {
-        const httpPort = process.env.AGENT_HTTP_PORT;
-        const adminPort = agentDto.adminApiPort || process.env.AGENT_ADMIN_PORT;
+        const httpPort = parseInt(process.env.AGENT_HTTP_PORT, 10);
```

Comment:
we use parseInt({}, 10) a lot, especially around env.  I wonder if its worth encapsulating this to ensure correctness.  Something to think about in the future.

Reply:
Ya, and the truth is the ,10 part is unnecessary and we only do it because of the linting rules, perhaps we should change the linting rules...